models/prices/ethereum/prices_ethereum_tokens.sql

```diff
@@ -89,6 +89,7 @@ FROM
     ("crv-curve-dao-token", "ethereum", "CRV", "0xd533a949740bb3306d119cc777fa900ba034cd52", 18),
     ("ctxc-cortex", "ethereum", "CTXC", "0xea11755ae41d889ceec39a63e6ff75a02bc1c00d", 18),
     ("cusdt-compound-usdt", "ethereum", "cUSDT", "0xf650c3d88d12db855b8bf7d11be6c55a4e07dcc9", 8),
+    ("cusdc-compound-usdc", "ethereum", "cUSDC", "0x39aa39c021dfbae8fac545936693ac917d5e7563", 8),
```

Comment:
Hi @fahimahmedx, it seems the token-id `cusdc-compound-usdc` is currently not available on coinpaprika API.
https://api.coinpaprika.com/v1/coins/cusdc-compound-usdc

Reply:
oops, just fixed.